spec/project_configuration_spec.rb

```diff
@@ -109,6 +109,28 @@
     end
   end
 
+  describe '#path' do
+    context 'when the path is not given in the constructor' do
+      it 'return the project name' do
+        config = Liftoff::ProjectConfiguration.new({
+          :project_name => 'My Cool Project'
+        })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.